Sources/TracingOpenTelemetrySemanticConventions/SpanAttributes+RPCSemantics.swift

```diff
@@ -209,7 +213,7 @@ public struct RPCAttributes: SpanAttributeNamespace {
             public static let unauthenticated = StatusCode(rawValue: 16)
 
             public func toSpanAttribute() -> SpanAttribute {
-                .int(self.rawValue)
+                SpanAttribute.int64(self.rawValue)
```

Comment:
Out of curiosity. Did `.int64` not work here?

Reply:
It would have worked I think